### ISSUE TITLE
Keyboard nav to end of page fails with non-text blocks

### ISSUE THREAD

**Issue by bschlagel** [hyperlink-academy/leaflet#162]
If a non-text block is the last block on the page, and I'm editing blocks above it, I expect cmd-down to nav me to the end of the page.

It appears to partially work (the last block gets the little selected state border) but the block is not actually selected b/c my cursor's still in its starting location; if I it return I'm in a new block where I started, not at the end of the page as expected.

Navigating by hitting down arrow til I'm actually at the block, then return, does work.

(Same issue with cmd-up / blocks at top of page, just rare to hit that case in practice.)
